packages/flame/lib/src/ai/README.md

```diff
@@ -0,0 +1,189 @@
+
+Most of the code in this directory was ported from [gdxAI] project by libGDX, licensed under
```

Comment:
Maybe this should be a license file instead of a readme

Reply:
It can't be in the LICENSE file, because then it would apply to all code in the directory and make it seem as if the code is owned by libGDX. 

So what we could do instead is to create the directory `flame/licenses/`, and put that license file there, together with the readme explaining how it applies.